imports/plugins/included/connectors-shopify/client/settings/shopify.html

```diff
@@ -44,7 +46,8 @@ <h4><span data-i18n="admin.shopifyConnectSettings.headingSync">Sync Settings</sp
     {{else}}
       <form data-event-action="setupShopifySync">
         <div>
-          <input type="checkbox" name="orders/create:updateInventory" {{integrationIsActive "orders/create:updateInventory"}}/>
+          <input type="checkbox" name="webhooks:orders/create:updateInventory" {{integrationIsActive
```

Comment:
@zenweasel orders/create:updateInventory - option here

Reply:
That's for incoming order from Shopify isn't is?